integration-test/src/main/java/org/apache/storm/st/topology/TestableTopology.java

```diff
@@ -17,14 +17,18 @@
 
 package org.apache.storm.st.topology;
 
+import java.util.concurrent.TimeUnit;
 import org.apache.storm.generated.StormTopology;
 
 public interface TestableTopology {
     String DUMMY_FIELD = "dummy";
-    //Some tests rely on reading the worker log. If emits are too close together and too much is logged, the log might roll, breaking the test.
-    int MIN_SLEEP_BETWEEN_EMITS_MS = 10;
-    int MAX_SLEEP_BETWEEN_EMITS_MS = 100;
+    int TIMEDATA_SLEEP_BETWEEN_EMITS_MS = 20;
+    //Some tests rely on reading the worker log. If there are too many emits and too much is logged, the log might roll, breaking the test.
```

Comment:
I wonder if we could instead configure the logs to roll less often, since that is the real issue?

Reply:
Yes, we could disable log rolling by customizing the cluster's `log4j2.xml`. I wanted to avoid doing that because it means you can't (reliably) run the integration test against your own cluster without having to edit `log4j2.xml` as well.

Reply:
OK, I assumed there was a way for the test itself to manipulate this. If not, then no worries.